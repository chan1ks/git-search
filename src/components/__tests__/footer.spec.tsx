import React from 'react';

import renderer from 'react-test-renderer';
import Footer from '../__layout/footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer author="chan1ks" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
