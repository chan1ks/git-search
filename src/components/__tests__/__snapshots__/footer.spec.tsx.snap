// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer>
  2020
   | Made with ‚ù§ by
   
  <a
    href="https://github.com/chan1ks"
  >
    chan1ks
  </a>
   | Built with
   
  <a
    href="https://www.gatsbyjs.com"
  >
    Gatsby
  </a>
</footer>
`;
